notification-service/src/test/java/com/redhat/parodos/notification/service/NotificationRecordServiceTests.java

```diff
@@ -17,6 +17,10 @@
 
 import com.redhat.parodos.notification.enums.Operation;
 import com.redhat.parodos.notification.enums.State;
+import com.redhat.parodos.notification.exceptions.NotificationRecordNotFoundException;
+import com.redhat.parodos.notification.exceptions.SearchByStateAndTermNotSupportedException;
+import com.redhat.parodos.notification.exceptions.UnsupportedStateException;
+import com.redhat.parodos.notification.exceptions.UsernameNotFoundException;
```

Comment:
Can you please remove unused `import static org.mockito.BDDMockito.given`?

Reply:
Done